### PR TITLE
MNT Don't expect trailing slash in tests

### PR DESCRIPTION
**Pull request by GuySartorelli**
https://github.com/silverstripe/silverstripe-framework/pull/10538 makes various methods _not_ include a trailing slash by default. This PR updates relevant tests to not expect trailing slashes.

## Parent issue
- https://github.com/silverstripe/silverstripe-cms/issues/2780

### CODE REVIEW

**Review by emteknetnz — COMMENTED** [review]
Approved pending the unit-tests being resolved once the framework PR is merged

**Comment by GuySartorelli** [comment]
See installer CI run re: failing unit tests

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**
